src/Struct/AgenticCommerceV1/PayPalAgentError.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Shopware\PayPalSDK\Struct\AgenticCommerceV1;
+
+use Shopware\PayPalSDK\Struct\Struct;
+
+/**
+ * @internal
+ */
+class PayPalAgentError extends Struct
```

Comment:
We are already in a PayPal context
```suggestion
class AgentError extends Struct
```